operations/logging.rst

```diff
@@ -0,0 +1,85 @@
+.. _logging:
```

Comment:
Out of curiosity only: what does this do?

Reply:
Creates a Sphinx label. Anywhere in the docs we can `:ref:'logging´` and Sphinx will know it refers to this file, wherever it is located. See https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#ref-role